### PR TITLE
chore: updating JS intros to use Quincy's approved copy

### PR DESCRIPTION
**Pull request by jdwilkin4**
## Summary of changes

This PR is responsible for updating the JS intros to use Quincy's approved copy. 

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -315,126 +316,129 @@
...
         ]
       },
       "learn-basic-javascript-by-building-a-role-playing-game": {
         "title": "Learn Basic JavaScript by Building a Role Playing Game",
         "intro": [
-          "JavaScript is a scripting language you can use to make web pages interactive. It is one of the core technologies of the web, along with HTML and CSS, and is supported by all modern browsers.",
-          "In this practice project, you'll learn fundamental programming concepts in JavaScript by building a role playing game. You'll learn to work with arrays, strings, objects, functions, loops, if/else statements, and more."
+          "JavaScript is a powerful scripting language that you can use to make web pages interactive. It's one of the core technologies of the web, along with HTML and CSS. All modern browsers support JavaScript.",
+          "In this practice project, you'll learn fundamental programming concepts in JavaScript by coding your own Role Playing Game. You'll learn how to work with arrays, strings, objects, functions, loops, if/else statements, and more."
```
Comment (naomi-lgbt) [inline client/i18n/locales/english/intro.json:347]:
```suggestion
          "In this practice project, you'll learn fundamental programming concepts in JavaScript by coding your own Role Playing Game. You'll learn how to work with arrays, strings, objects, functions, loops, <code>if/else</code> statements, and more."
```

These are technically tokens so should not be translated.

File: client/i18n/locales/english/intro.json [hunk 2]
```diff
@@ -315,126 +316,129 @@
...
         ]
       },
       "learn-localstorage-by-building-a-todo-app": {
         "title": "Learn localStorage by Building a Todo App",
         "intro": [
-          "Local storage is a web browser feature that allows web applications to store key-value pairs persistently within a user's browser, enabling data to be saved and retrieved across page sessions.",
-          "In this TODO application, you will learn how to handle form inputs, manage local storage, perform CRUD (Create, Read, Update, Delete) operations on tasks, implementing event listeners, and toggling UI elements."
+          "Local storage is a web browser feature that lets web applications store key-value pairs persistently within a user's browser. This allows web apps to save data during one session, then retrieve it in a later page session.",
+          "In this TODO application, you'll learn how to handle form inputs, manage local storage, perform CRUD (Create, Read, Update, Delete) operations on tasks, implementing event listeners, and toggle UI elements."
```
Comment (naomi-lgbt) [inline client/i18n/locales/english/intro.json:420]:
```suggestion
          "In this TODO application, you'll learn how to handle form inputs, manage local storage, perform CRUD (Create, Read, Update, Delete) operations on tasks, implement event listeners, and toggle UI elements."
```

This shouldn't be a gerund.

File: client/i18n/locales/english/intro.json [hunk 3]
```diff
@@ -315,126 +316,129 @@
...
         "intro": [
-          "Mastering essential string and array methods like the <code>find()</code>, <code>forEach()</code>, <code>map()</code>, and <code>join()</code> methods is crucial for developing dynamic web applications.",
-          "In this project you will build out a basic MP3 player using HTML, CSS, and JavaScript. The project covers fundamental concepts such as handling audio playback, managing a playlist, implementing play, pause, next, previous, and shuffle functionalities, as well as dynamically updating the user interface based on the current song."
+          "Now let's learn some essential string and array methods like the <code>find()</code>, <code>forEach()</code>, <code>map()</code>, and <code>join()</code>. These methods are crucial for developing dynamic web applications.",
+          "In this project, you'll code a basic MP3 player using HTML, CSS, and JavaScript. The project covers fundamental concepts such as handling audio playback, managing a playlist, implementing play, pause, next, previous, and shuffle functionalities. You'll even learn how to dynamically update your user interface based on the current song."
```
Comment (naomi-lgbt) [inline client/i18n/locales/english/intro.json:441]:
```suggestion
          "In this project, you'll code a basic MP3 player using HTML, CSS, and JavaScript. The project covers fundamental concepts such as handling audio playback, managing a playlist, implementing play, pause, next, and previous functionalities. You'll even learn how to dynamically update your user interface based on the current song."
```

Ignore this if I'm wrong, but I thought we scrapped the shuffle feature.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: client/i18n/locales/english/intro.json (hunk 3 above)
Comment (jdwilkin4) [reply client/i18n/locales/english/intro.json:441]: just checked and shuffle was added in 👍

**Review by Ksound22 — APPROVED** [review]
This is good to go in

**Review by naomi-lgbt — APPROVED** [review]

**Review by moT01 — APPROVED** [review]

**Pull request merged**